### PR TITLE
Fix the Function interface (#11269)

### PR DESCRIPTION
**Pull request by falsandtru**

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
What's the bug tracking this change?

**Comment by falsandtru** [comment]
Fixes #11269

File: lib/lib.d.ts [hunk 1]
```diff
@@ -260,6 +260,8 @@ interface Function {
       */
     bind(this: Function, thisArg: any, ...argArray: any[]): any;
 
+    toString(): string;
```
Comment (mhegazy) [inline lib/lib.d.ts]: can you add a comment here:

```ts
  /** Returns a string representation of an object. */
  toString(): string;
```

Comment (falsandtru) [inline lib/lib.d.ts]: fixed.

**Pull request merged**